hack/deployer/config/plans.yml

```diff
@@ -105,7 +105,7 @@ plans:
   provider: eks
   machineType: c5d.2xlarge
   serviceAccount: false
-  kubernetesVersion: 1.17
+  kubernetesVersion: 1.18
```

Comment:
Default version seems to be `1.20.9`, any reason to not use it ? (also I can't see `1.18` in the list)
![image](https://user-images.githubusercontent.com/976373/143208579-75f577bc-ad7c-4ede-a5c9-dfd400dc8192.png)

Reply:
I was looking at https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html. +1 to use `1.20`.